src/Microsoft.ML.TensorFlow/TensorflowTransform.cs

```diff
@@ -36,21 +37,27 @@ public sealed class TensorFlowTransform : ITransformer, ICanSaveModel
     {
         public sealed class Arguments : TransformInputBase
         {
+            [Argument(ArgumentType.Required, HelpText = "TensorFlow model used by the transform. Please see https://www.tensorflow.org/mobile/prepare_models for more details.", SortOrder = 0)]
+            public string Model;
 
-            [Argument(ArgumentType.Required, HelpText = "This is the frozen protobuf model file. Please see https://www.tensorflow.org/mobile/prepare_models for more details.", ShortName = "model", SortOrder = 0)]
-            public string ModelFile;
+            [Argument(ArgumentType.AtMostOnce, HelpText = "Indicator for frozen models", ShortName = "frozen", SortOrder = 1)]
+            public bool IsFrozen = TensorFlowEstimator.Defaults.IsFrozen;
```

Comment:
>IsFrozen [](start = 24, length = 8)

Is there a way to figure this out from the Model argument? For example, check if it is a directory or a file, or try loading it in a try/catch?

Reply:
Yes. Added a utility method TensorFlowUtils.IsFrozenTensorFlowModel()  to do this check.  

(will delete the commented out code here in the next push)

---
In reply to: [217436951](https://github.com/dotnet/machinelearning/pull/853#discussion_r217436951) [](ancestors = 217436951)